intellij-toml/src/main/resources/META-INF/plugin.xml

```diff
@@ -18,43 +18,5 @@
 
     <depends>com.intellij.modules.lang</depends>
```

Comment:
I'm not sure if we need to have this dependency here

Reply:
We need it. Otherwise, we can't rely on that IDE contains classes required for language support. For example, Code With Me client doesn't have `com.intellij.modules.lang` module. As a result, you can't install Toml plugin on CWM client and it's desired behavior

Reply:
Oh, probably I misunderstood you. We actually don't need this `depends` here because we have it in `toml-core.xml`

Reply:
Then probably the same can be applied to `plugin.xml` for intellij-rust

Reply:
Yep. Would you like to fix it in separate PR?

Reply:
Let's drop it not to produce `[gradle-intellij-plugin :intellij-toml:verifyPlugin] Duplicated dependency on 'com.intellij.modules.lang'` warning - we have necessary dependency declaration in core xml